### ISSUE TITLE
Bug allowing player to crash movecraft with proper timing.

### ISSUE THREAD

**Issue by danielcbailey** [msummers123/Movecraft-3#171]
There is a small bug that allows players to crash movecraft while moving. The craft is piloted with a chest with an easy to break block next to the chest. Start cruising and get ready to preform the following: Right before the craft moves, replace the easy to break with another chest to make a large chest. Congratulations, you just crashed movecraft.

**Comment by keough99** [comment]
Hopfully this isn't in older versions

**Comment by TylerS1066** [comment]
Most likely it's in all versions

**Comment by msummers123** [comment]
Yeah, I'm afraid I'm not sure how to fix this exactly, short of checking for linked chests every time a block is placed (which would be very laggy).
